nautobot/extras/models/customfields.py

```diff
@@ -589,6 +585,16 @@ def to_form_field(
 
         return field
 
+    def to_filter_form_field(self, *args, **kwargs):
+        """Return a filter form field suitable for filtering a CustomField's value for an object."""
+        form_field = self.to_form_field(*args, **kwargs)
+        if self.type == CustomFieldTypeChoices.TYPE_SELECT:
+            # Remove the blank choice, as StaticSelect2Multiple do not need a blank choice
+            # Its blank by default
+            choices = form_field.choices[1:]
+            form_field.widget = StaticSelect2Multiple(choices=choices)
```

Comment:
This feels wrong to me, actually - for TYPE_SELECT we're using a MultiValueCharFilter and supporting all of the FILTER_CHAR_BASED_LOOKUP_MAP filter variants (`__isw`, `__re`, etc.) - so from a form perspective shouldn't this be a MultiValueCharField ratehr than a StaticSelect?

Reply:
Attempting to assert your concern here: `MultiValueCharField` is does not inherit from `ChoiceField` and therefore cannot accept choices.